pages/1.13/security/oss/authentication/index.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: layout.pug
+navigationTitle:  Authentication
+title: Authentication
+excerpt: Authenticating users against DC/OS
+menuWeight: 30
+
+---
+
+<!-- The source repository for this topic is https://github.com/dcos/dcos-docs-site -->
+
+# Authentication in DC/OS
+
+In DC/OS user authentication is enabled by default.
+
+Every user that wants to perform an operation on a DC/OS cluster (other than login) must first be authenticated.
+
+DC/OS handles user authentication decentralized by the means of authentication tokens. Authentication tokens are distributed by the Identity and Access Manager (IAM) on a per user basis. The tokens can be verified by the IAM, but also out-of-band by any 3rd-party entity. Allowing token verification to happen independent of the IAM is what makes this approach highly scalable in comparison to centralized session state keeping. On the other hand with tokens user authentication state cannot easily be revoked.
```

Comment:
> The tokens can be verified by the IAM

It does not have code for doing that. It only emits authentication tokens.

Reply:
Which is what the sentence "In Open DC/OS the only authenticator in the system is Admin Router." below says, too! :-)